app/mainAppWindow/index.js

```diff
@@ -13,11 +13,10 @@ let aboutBlankRequestCount = 0;
 
 let window = null;
 
-exports.onAppReady = async function onAppReady() {
+exports.onAppReady = function onAppReady() {
 	window = createWindow();
-	await clearCache(window.webContents.session);
-	await clearStorageData(window.webContents.session);
-
+	window.webContents.session.clearCache(()=> console.log("session cache cleared"));
```

Comment:
We need to remove this and the next line. They where PoC that didn't solve the issue. Thanks!

Reply:
Reverted commit b330962 and the extra line of logging.